src/assets/scss/_recordset.scss

```diff
@@ -1,155 +1,197 @@
 @import "helpers";
 @import "variables";
 
-.recordset-container {
-  .top-panel-container {
-    .top-flex-panel .top-left-panel div.panel-header {
-      bottom: 10px;
+.chaise-body {
```

Comment:
It's hard for me to figure out what exactly you've changed. If you're relying on component to pass the default width values, why are you setting them in css as well?

By the way, just like @jrchudy mentioned, the same set of rules can be found in [app.scss](https://github.com/informatics-isi-edu/chaise/blob/react/src/assets/scss/app.scss#L443). The idea is that the one in app.scss is supposed to define the "default "values. And then in here we customized these rules to make it look better in case of recordset.

If the changes in here was supposed to be for the "default" values, you should change the app.scss one instead.

Reply:
the difference is that I an setting the value of flex-basis as unset in the _recordset.scss file as compared to it being flex-basis: $left-panel-width-lg; in app.scss

Reply:
this is necessary as giving the element a flex-basis value interferes with the resizable filter's working

Reply:
OK I just wanted to confirm that the rules in `app.scss` are still necessary.